__tests__/app.test.js

```diff
@@ -28,6 +27,26 @@ describe("GET /api/topics", () => {
   });
 });
 
+describe("GET /api/articles/:article_id", () => {
+  test("status 200, should return an article object, which should have the following properties: author ,title ,article_id ,body ,topic ,created_at ,votes ,article_img_url", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article.article_id).toBe(2);
+      });
+  });
+  test("status 404, should return obj with message of not found", () => {
+    return request(app)
+      .get("/api/articles/442")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not Found");
+      });
+  });
```

Comment:
There is another possibility we should be testing for. What if the user tries to use an invalid id like `/api/articles/ten` instead of `10`  What kind of error status should we be expecting for this?